### PR TITLE
add the sym_not / full operator to support dynamic shape

### PR DESCRIPTION
**Pull request by lanluo-nvidia**
# Description

Add sym_not / full operator to support dynamic shape

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py [hunk 1]
```diff
@@ -1759,6 +1759,30 @@ def aten_ops_logical_not(
     )
 
 
+# TODO: confirm with Dheeraj, whether it requires supports_dynamic_shapes=True
+# according to https://pytorch.org/docs/stable/generated/torch.sym_not.html
+# Boolean value of Tensor with more than one value is not allowed, so it must be static_shaped
+# eg. allowed: True or torch.tensor(True) or torch.tensor([False]) or torch.tensor([[False]])
+# eg. not allowed: torch.tensor([True, False]) or torch.tensor([[True], [False]])
```
Comment (peri044) [inline py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py:1766]: You can enable supports_dynamic_shapes=True in this case so that there's no segmentation. Most likely, there wouldn't be a scenario where input is a dynamic list for this op (your code throws an error here anyways, so that's fine).

**Review by peri044 — APPROVED** [review]
LGTM
